platform/ios/docs/guides/UserInteractions.md

```diff
@@ -0,0 +1,47 @@
+# User Interactions in the Mapbox iOS SDK
+
+The Mapbox iOS SDK provides a set of built-in gesture recognizers to developers. These interactions can be customized to fit your use case. You see what gesture recognizers are on your `MGLMapView` by accessing the `gestureRecognizers` property on your map.
```

Comment:
Remove “to developers”: the built-in gesture recognizers are primarily provided for the benefit of end users, not developers.